src/components/preferences/General.js

```diff
@@ -191,6 +193,42 @@ function General(props) {
                         label={t("urlImportEmailLbl")}
                     />
                 </Grid>
+                <Grid item>
+                    <Field
+                        id={buildID(
+                            baseId,
+                            ids.PERIODIC_EMAIL_NOTIFICATION_SWITCH
+                        )}
+                        component={FormSwitch}
+                        name={
+                            prefConstants.keys
+                                .ENABLE_PERIODIC_EMAIL_NOTIFICATION
+                        }
+                        color="primary"
+                        inputProps={{
+                            "aria-label": t("periodicEmailLbl"),
+                        }}
+                        label={t("periodicEmailLbl")}
+                    />
+                </Grid>
+                <Grid item xs={6}>
+                    <Field
+                        id={buildID(baseId, ids.PERIODIC_NOTIFICATION_PERIOD)}
+                        name={prefConstants.keys.PERIODIC_NOTIFICATION_PERIOD}
+                        label={t("periodicPeriodLbl")}
+                        component={FormSelectField}
```

Comment:
I've been preferring `FormTextField` with the `select` prop, for a more consistent UX (such as using an outlined form field):
```javascript
                        component={FormTextField}
                        select
```

Reply:
Ah, interesting. I saw that was in use one place and the select in a different one, and went with FormSelectField because it seemed like it should be more usual, but I'm good to change it as well, so I'll do that.